### ISSUE TITLE
read_pandas does not correctly handle multiline strings

### ISSUE THREAD

**Issue by fabiencelier** [atoti/atoti#186]
## Field containing `\n`

```python
import pandas as pd, atoti as tt
session = tt.create_session()
df = pd.DataFrame({"a": [1,2,3,4], "b": ["b1", "b\n2", "b3", "b4"]})
store = session.read_pandas(df, store_name="multi n")
store.head()
```
![first rows of atoti store](https://user-images.githubusercontent.com/17087544/99243928-aac44580-2801-11eb-9b9d-587f3bf13a51.png)

This issue is solved by adding `process_quotes=True` which is passed to the read_csv method but this should be done automatically.

## Field containing `\r`

```python
import pandas as pd, atoti as tt
session = tt.create_session()
df = pd.DataFrame({"a": [1,2,3,4], "b": ["b1", "b\r2", "b3", "b4"]})
store = session.read_pandas(df, store_name="multi r")
store.head()
```

![Screenshot from 2020-11-16 11-50-36](https://user-images.githubusercontent.com/17087544/99244142-f7a81c00-2801-11eb-888d-b002fb583cf7.png)

This is because pandas does not escape the field containing `\r` and atoti consider it as a new line character:

``` python
> df.to_csv(index=False)
'a,b\n1,b1\n2,b\r2\n3,b3\n4,b4\n'
```

**Comment by fabiencelier** [comment]
I don't know how to force the escape when there is a `\r`.
Related StackOverflow question: https://stackoverflow.com/q/64857438/4678661

**Comment by tibdex** [comment]
Fixed in atoti 0.6.0 since `read/load_pandas()` don't use an intermediate CSV representation anymore.
